timely_beliefs/__init__.py

```diff
@@ -1,12 +1,12 @@
 # flake8: noqa
 
-from timely_beliefs.sensors.classes import Sensor, DBSensor
-from timely_beliefs.sources.classes import BeliefSource, DBBeliefSource
+from timely_beliefs.sensors.classes import DBSensor, Sensor  # isort:skip
+from timely_beliefs.sources.classes import BeliefSource, DBBeliefSource  # isort:skip
```

Comment:
Why are these skipped?

Reply:
The import order is very important here. The utils need these high-level classes, but isort would put them on top, due some sort order (I believe alphabetical)

Reply:
isort has its sorting assumption (I believe alphabetical). In this case, itś crucial that the main classes are there first, as the utils rely on them.